example/spec/helpers/workflowUtils.js

```diff
@@ -273,9 +274,28 @@ async function buildAndStartWorkflow(
   buildWorkflow(stackName, bucketName, workflowName, collection, provider, payload, meta);
   return startWorkflow(stackName, bucketName, workflowName, workflowMsg);
 }
+
```

Comment:
The function is moved to here from example/spec/parallel/orca/OrcaBackupAndRecoverySpec.js